elytron/src/test/resources/org/wildfly/extension/elytron/credential-security-factories.xml

```diff
@@ -16,6 +16,7 @@
                                    server="false"
                                    obtain-kerberos-ticket="true"
                                    debug="true"
+                                   mechanisms="KRB5 KRB5LEGACY"
```

Comment:
@darranl Do we prefer one or two standalone attributes for this?
I am also not sure if allowedValues would be reasonable, as the list would duplicate content of property file in elytron and the same should be added into x500-attribute-principal-decoder/attribute-name.